Update active-directory-aadconnect.md

RobdeJong opened the pull request:
> [!IMPORTANT]
> Microsoft doesn't support modifying or operating Azure AD Connect sync outside of the actions that are formally documented. Any of these actions might result in an inconsistent or unsupported state of Azure AD Connect sync. As a result, Microsoft can't provide technical support for such deployments.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @eslesar, has been notified to review your proposed change.

MGoedtel commented:
@4c74356b41 thank you for taking the time to submit a request to make corrections/updates to this article.  I have reviewed them and approve them to be merged.

RobdeJong committed:
Update active-directory-aadconnect.md

> [!IMPORTANT]
> Microsoft doesn't support modifying or operating Azure AD Connect sync outside of the actions that are formally documented. Any of these actions might result in an inconsistent or unsupported state of Azure AD Connect sync. As a result, Microsoft can't provide technical support for such deployments.
